code/modules/antagonists/changeling/changeling.dm

```diff
@@ -399,6 +399,17 @@
 		to_chat(owner.current, span_warning("We lack the energy to evolve new abilities right now!"))
 		return FALSE
 
+	if((sting_path == /datum/action/changeling/sting/transformation) && (purchased_powers[/datum/action/changeling/sting/false_armblade]))
+		to_chat(owner.current, span_notice("False changeling sting is now available."))
+	else if((sting_path == /datum/action/changeling/sting/false_armblade) && (purchased_powers[/datum/action/changeling/sting/transformation]))
+		to_chat(owner.current, span_notice("False changeling sting is now available."))
+
+	//Pre-requisite power for false ling helper ability
+	if(sting_path == /datum/action/changeling/sting/fake_changeling)
+		if(!(purchased_powers[/datum/action/changeling/sting/transformation]) || !(purchased_powers[/datum/action/changeling/sting/false_armblade]))
+			to_chat(owner.current, span_warning("We lack the pre-requisite abilities to evolve fake changeling sting."))
+			return FALSE
```

Comment:
This hardcoding is very gross, and also completely ignores the whole `prereq_abilitiy` list.

Reply:
Way around this likely requires you `new` the ability before going into `give_power`

```dm
var/datum/action/changeling/power = new sting_path()
if(!power.can_give(src))
	qdel(power)
	return FALSE
```
```dm
/datum/action/changeling/proc/can_give(datum/antagonist/changeling/ling)
	for(pre_req_power in prereq_abilitiy)
		if(!ling.purchased_powers[pre_req_power])
			return FALSE
```